jkube-kit/build/service/buildpacks/src/main/java/org/eclipse/jkube/kit/service/buildpacks/BuildPackCommand.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.service.buildpacks;
+
+import org.eclipse.jkube.kit.common.ExternalCommand;
+import org.eclipse.jkube.kit.common.KitLogger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class BuildPackCommand extends ExternalCommand {
+  private final String packCliPath;
+  private final List<String> commandLineArgs;
+  private final StringBuilder errorBuilder;
+  private final Consumer<String> commandOutputConsumer;
+
+  public BuildPackCommand(KitLogger log, String packCliPath, List<String> cmdArgs, Consumer<String> outputConsumer) {
```

Comment:
`String packCliPath`

It might make more sense to pass in a File or a Path `Path packCli`